PowerShell/update-deployment-settings.ps1

```diff
@@ -119,6 +120,24 @@
                             Write-Host "Environment variable $configurationVariableName is Null or Empty"
                         }
                     }
+                    #Set WebHook URL variables
+                    elseif($configurationVariableName.StartsWith("webhookurl.", "CurrentCultureIgnoreCase")) {
+                        if(-not [string]::IsNullOrWhiteSpace($configurationVariableValue))
+                        {
+                            $schemaName = $configurationVariableName -replace "webhookurl.", ""
+                            #$endPointResults =  Get-CrmRecords -conn $conn -EntityLogicalName "serviceendpoints" -FilterAttribute "name" -FilterOperator "eq" -FilterValue $schemaName -Fields "name"
```

Comment:
@RajeevPentyala Can you remove the unused / commented code? May be confusing to anyone using our code as a reference as it makes it less readable.